### DISCUSSION TITLE
Looking for the output

### DISCUSSION THREAD

**Discussion by MaximumCrafter** [neonbjb/tortoise-tts#295] [Q&A] [answered]
Hello! So I was trying to generate some audio. I got it to generate, but I cant seem to find where it gets put. Is there a specific place that it should be outputting? Or is it different for each person? Apologies if the answer is simple, I've looked but I can't find it anywhere.

**Comment by Pandaily591** [comment, marked as answer]
Yeah, you should find results in tortoise-tts\tortoise-tts\results

it uses the same names for the 3 output candidates each time, so it will overwrite the last batch.

**Reply by MaximumCrafter** [reply]
Thank you so much!

**Comment by jnberger** [comment]
How do you change the output naming and location?
